src/libs/ajax.js

```diff
@@ -479,6 +479,11 @@ const Groups = signal => ({
 
       setPolicy: (policyName, value) => {
         return fetchSam(`${resourceRoot}/policies/${policyName}/public`, _.mergeAll([authOpts(), { signal, method: 'PUT' }, jsonBody(value)]))
+      },
+
+      isMember: async () => {
+        const res = await fetchSam(`${resourceRoot}/action/use`, _.merge(authOpts(), { signal }))
+        return res.json()
```

Comment:
It would technically work (and is allowed by the standard) if the endpoint returns a boolean value as the response:
```javascript
> JSON.parse('true') ? 'yes' : 'no'
'yes'
> JSON.parse('false') ? 'yes' : 'no'
'no'
```
Because it's not a common pattern, it would not surprise me if the encoding library screwed this up. For the record, Node does the right thing:
```javascript
> JSON.stringify(false)
'false'
```